### DISCUSSION TITLE
How can I solve this shadow?

### DISCUSSION THREAD

**Discussion by wj8400684** [amwx/FluentAvalonia#310] [Q&A]
<img width="1068" alt="e5b3112fda59eb977ec84ff8742b786" src="https://user-images.githubusercontent.com/39239954/221151446-37ff70ad-d029-4e80-b757-ce750341f558.png">

<img width="1848" alt="4e2cab971ae17ebd6c523ea9e2e2e22" src="https://user-images.githubusercontent.com/39239954/221152114-dd9c5b49-4468-4d29-b1f6-4562423d87b3.png">

**Comment by amwx** [comment]
Unfortunately, that's not possible with `AppWindow` (this is limitations with DWM). If you want to do custom things with the window like that, you'll have to stick with a regular Avalonia Window
